### DISCUSSION TITLE
How to set up Social Media meta tags for Voila Dashboards hosted as dockerized container (Open Graph Protocol)

### DISCUSSION THREAD

**Discussion by BrainWave89** [voila-dashboards/voila#1351] [Q&A]
Hi,

I am building a web app in Python using Jupyter Lab, Voila, and Docker. I use Google Cloud Run to deploy the dockerized container and I'd like to set up social media tags (specifically the ones listed below).  

Here are a few observations and things that I've tried already:
1) The Python file name (.ipynb) and URL show up by default when sharing the link - which is nice. 
2) I've tried modifying the page.html file in the share/jupyter/voila/template/base directory. When running Voila locally, the page.html content is being displayed as text file pop up window but it does not seem to have any effects on the application. This tells me that the correct page.html file is being loaded (i.e. I am working with the correct template).

It would be great if someone could explain how setting up social media tags is done correctly. Thank you so much and looking forward to hearing back from the Voila dashboard community.

Social media tags:
 meta property 'og:title' content='Title of the article'/>

meta property 'og:image' content '//media.example.com/ 1234567.jpg'/>

meta property'og:description' content='Description that will show in the preview'/>

meta property'og:url' content='//www.example.com/URL of the article'/>

**Comment by jtpio** [comment]
@BrainWave89 maybe this could be achieved with a custom template, by adding these extra tags to the template by default?

Here is an example of a template that could be useful for inspiration: https://github.com/voila-dashboards/voila-material
For setting social media tags the template would likely be smaller with probably only the `html_head` block to override.

**Comment by ghost** [comment]
@jtpio How is a voila docker image built for cloud run? 

I've added the question here: https://github.com/voila-dashboards/voila/issues/1521
